Source/WebCore/dom/EventNames.cpp

```diff
@@ -24,7 +24,7 @@
 namespace WebCore {
 
 #define INITIALIZE_EVENT_NAME(name) \
-    name##Event(#name, AtomString::ConstructFromLiteral),
+    name##Event(#name ## _s),
```

Comment:
I totally would have merged this into one line.

Reply:
Ok